### PR TITLE
add basic confirm modal on delete

### PR DESCRIPTION
**Pull request by karissa**
I couldn't get the app running (a bug on create related to dat-encoding) to test it by hand so there probably is a bug. I wonder if anyone has time to finish it up?

### CODE REVIEW

**Comment by karissa** [comment]
OK got it running and now this is working and ready for styling!

**Comment by karissa** [comment]
https://www.dropbox.com/s/by0d0zh46hzcmom/deleteconfirm.mov?dl=0

File: models/repos.js [hunk 1]
```diff
@@ -50,14 +51,28 @@ function createModel (cb) {
...
   model.effect('delete', (state, data, send, done) => {
     const dat = data
-    manager.remove(dat, done)
+    const encodedKey = encoding.encode(dat.key)
+    send('location:set', `?delete=${encodedKey}`, done)
+  })
+
+  model.effect('deleteConfirm', (state, data, send, done) => {
+    const link = data
+    console.log(data)
```
Comment (yoshuawuyts) [inline models/repos.js]: There's an extra console.log here

Comment (Kriesse) [inline models/repos.js]: removed it

**Comment by yoshuawuyts** [comment]
cool, rebased!

**Comment by karissa** [comment]
Woot!

**Pull request merged**